pattern-1/artifacts/routes/wso2apim-analytics-route.yaml

```diff
@@ -19,11 +19,22 @@ metadata:
   labels:
     pattern: wso2apim-pattern-1
 spec:
-  host: wso2apim-analytics
+  host: analytics.customer.poc.wso2.com
```

Comment:
Might be better to use a more generic hostname here without using "customer.poc".

Reply:
Sure. Thought of having this only for poc made me use a genric name as customer+poc.